modules/learn/pages/data/distributed-transactions.adoc

```diff
@@ -0,0 +1,315 @@
+
+The indexes provided by the Index, Search, and Analytics services are not synchronously updated with the commits performed by transactions, and instead they are updated with _Eventual Consistency_. Hence, a query performed immediately after committing a transaction may not see the effects of the transaction. 
+
+The Query Service provides the transactional scan consistency parameter, `request_plus`, which allows queries to wait for indexes to be appropriately updated, following a transaction. This `request_plus` parameter ensures that your queries operate on the latest visible data.
+When a query is used inside a transaction, the transactional scan consistency is set to `request_plus` by default, and hence ensures that the query will see all the committed changes. 
+
+Note that you can choose to update the scan consistency level to `not_bounded` in some cases such as the following:
+
+* If your query uses USE KEYS.
+* If you know that the data being accessed or consumed by the transaction has not been recently updated.
+* If your transaction does not care about the latest data, for example UPSERT or INSERT statements.
+
+=== Transactions and Replication (XDCR)
+
+xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication] (XDCR) supports eventual consistency of transactional changes. No uncommitted changes will be ever sent to the target clusters.  Once committed, the transactional changes will arrive one by one at the target. If the connection is lost midway while receiving a transaction it is possible for the target to receive a partial transaction.
+
+* Transactionally modified documents should only be replicated across clusters if no transactions involving the same documents can occur on those clusters simultaneously in a bidirectional XDCR. 
+
+* Always follow the steps to xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc#ensuring_safe_failover[Ensure Safe Failover] for information on failing a transactional application from one data center to another.
+
+== Transaction Mechanics
+
+Consider the transaction example to transfer funds from Beth’s account to Andy’s account. 
+
+Assuming that the 2 documents involved in this transaction live in two different nodes, here are the high-level steps that the transaction follows:
+
+image::transaction-mechanics-steps.png["Transaction mechanics explaining the high-level steps that a transaction follows"]
+
+Each execution of the transaction logic in an application is called an 'attempt' inside the overall transaction.
+
+=== Active Transaction Record Entries
+
+The first mechanic is that each of these attempts adds an entry to a metadata document in the Couchbase cluster. These metadata documents are called _Active Transaction Records_, or ATRs. 
+ATRs are created and maintained automatically and are easily distinguishable by their prefix `_txn:atr-`. They are viewable and should not be _modified externally_.
```

Comment:
I think `_txn:atr-` will need putting in the headers with a {} key here, to stop the underscore being parsed as an asciidoc italic (unless Antora is smarter than GH's asciidoc renderer?).
Speaking of italics, should the 'should not' part of the statement be added into the italics, so covering 5 words in place of just the last two?